addons/project/data/mail_template_data.xml

```diff
@@ -3,7 +3,7 @@
     <data noupdate="1">
         <!-- Sample stage-related template -->
         <record id="mail_template_data_project_task" model="mail.template">
-            <field name="name">Task: Reception Acknowledgment</field>
+            <field name="name">Field Service: Reception</field>
```

Comment:
This is strange. I may use project for something else than field service, which is quite niche. Not my domain but I feel like a generic naming is better than a specific-but-often-wrong naming.